<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cells2d3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
       <item row="0" column="1" rowspan="2">
        <widget class="QGraphicsView" name="graphicsView">
         <property name="renderHints">
          <set>QPainter::Antialiasing</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>View</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="load_button">
            <property name="text">
             <string>Load Image...</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="zoomLayout">
            <item>
             <widget class="QPushButton" name="zoom_in_button">
              <property name="text">
               <string>Zoom In</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="zoom_out_button">
              <property name="text">
               <string>Zoom Out</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reset_view_button">
              <property name="text">
               <string>Reset View</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Channels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="list_channels"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_display_channel">
            <property name="text">
             <string>Display Channel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gamma (display brightness):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_gamma">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>ROI Definition</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="5" column="3">
           <widget class="QPushButton" name="save_area">
            <property name="text">
             <string>Save Area...</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="create_roi_button">
            <property name="text">
             <string>Create ROI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="depth_label">
              <property name="text">
               <string>depth [um]:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="pixel_um_label">
              <property name="text">
               <string>pixel/um:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="depth_input">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="pixel_um_input">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1" colspan="3">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="0">
             <widget class="QRadioButton" name="radio_linear">
              <property name="text">
               <string>Linear</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="radio_curved">
              <property name="text">
               <string>Curved</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Arial</family>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Bottom-Shape:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="edit_boundary_checkbox">
            <property name="text">
             <string>Edit Boundary</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="button_delete_boundary">
            <property name="text">
             <string>Delete Boundary</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="reset_roi_button">
            <property name="text">
             <string>Reset ROI</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
